class/sim800c/at_device_sim800c.c

```diff
@@ -767,6 +767,11 @@ static void sim800c_init_thread_entry(void *parameter)
             /* "CT" */
             LOG_I("%s device network operator: %s", device->name, parsed_data);
         }
+        else
+        {
+            AT_SEND_CMD(client, resp, 0, 300, "AT+CSTT");
```

Comment:
The AT command of ```CSTT``` can set APN, but ```AT+CSTT``` maybe is useless. 
Because you haven't set access point network(APN) as a parameter.

-----

Maybe we can delete this code if it is useless.

Reply:
Thanks for the reply, without setting this CMD event it is empty. The module refuses to work. 
Per the state machine in SIM800's manual, setting  "AT+CSTT" is necessary for the state to change to "IP START". It might be the case. 
![image](https://user-images.githubusercontent.com/3821068/115697193-3d41b580-a35b-11eb-9376-017ed05ec7c5.png)
Page  26
https://cdn-shop.adafruit.com/product-files/2637/SIM800+Series_TCPIP_Application+Note_V1.01.pdf

Reply:
OK，Thanks for your explain.